File: app/src/main/java/com/sleticalboy/doup/util/ToastUtils.java
package com.sleticalboy.doup.util;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by Android Studio.
 * Date: 10/29/17.
 *
 * @author sleticalboy
 */

public class ToastUtils {

    public static void showToast(Context context, CharSequence text) {
        if (context == null)
            throw new IllegalArgumentException("context was null");
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show();
    }

    public static void showToast(Context context, int resId) {
        showToast(context, context.getResources().getString(resId));
    }
}

File: my-utils/src/main/java/com/sleticalboy/util/ToastUtils.kt
package com.sleticalboy.util

import android.content.Context
import android.widget.Toast
import java.lang.IllegalArgumentException

/**
 * Created by Android Studio.
 * Date: 10/29/17.
 *
 * @author sleticalboy
 */
object ToastUtils {

    fun showToast(context: Context?, text: CharSequence) {
        if (context == null)
            throw IllegalArgumentException("context was null")
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show()
    }

    fun showToast(context: Context, resId: Int) {
        showToast(context, context.resources.getString(resId))
    }
}
